### PR TITLE
Fixes #33927 - make pr template a little less harsh

### PR DESCRIPTION
**Pull request by jlsherrill**
#### What are the changes introduced in this pull request?
Change the pr template ot use 4 hashes instead of 3
#### What is the thinking behind these changes?
i think the 3rd header is a little large/bold, especially when there isn't a lot of text
#### What are the testing steps for this pull request?
use your eyes

### CODE REVIEW

**Comment by jlsherrill** [comment]
Before:
image

**Comment by jlsherrill** [comment]
After:
image

**Comment by jlsherrill** [comment]
This may be personal preference, it kept irking me though :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-### What are the changes introduced in this pull request?
+#### What are the changes introduced in this pull request?
 
-### What is the thinking behind these changes?
+#### What is the thinking behind these changes?
```
Comment (jturel) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: In similar spirit, how about something like 'Considerations taken when implementing this change?' The current phrasing almost sounds accusatory ;)

Comment (jlsherrill) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: sounds good!

**Review by jturel — APPROVED** [review]
What _were_ you thinking with this change @jlsherrill ? :laughing: 

ACK!

**Comment by jlsherrill** [comment]
I never know!

**Pull request merged**